### DISCUSSION TITLE
Statistics graphs omit High and Low levels

### DISCUSSION THREAD

**Discussion by Navid200** [NightscoutFoundation/xDrip#4156] [Ideas]
The High and Low levels, which can be set under Settings, Glucose Units, are visible as two horizontal lines (red for Low and yellow for High) on the Ambulatory Glucose Profile (AGP) graph (Percentile chart).
But, it is not easy to see what those values are considering how large the scale of the vertical axis is.
Neither of the other two statistical pages show those values.

Some of us use those graphs to create a report to give to our Endocrinologist.
It would be great if the Low Value and the High Value figures were clearly shown on the AGP graph, and added to the list of parameters shown on the other statistics page.

**Comment by emp-00** [comment]
agreed, this make a lot of sense.

**Comment by doc77** [comment]
Numbers are displayed on the vertical axis, and are noted every 50 mg/dl. It does not seem so difficult to make an accurate estimation based on this.

**Comment by Navid200** [comment]
No, it's not hard to estimate.
However, since the real accurate values are present, why estimate?  Why not include them on the list of parameters reported already?
Considering that the High and Low levels directly affect PGS, why not report the exact values?

**Comment by Navid200** [comment]
Also, not everyone uses mg/dL.

This is a snapshot of the code that creates the grid.

![Screenshot 2021-02-26 231108](https://user-images.githubusercontent.com/51497406/109375084-f74f0080-7887-11eb-89c0-9a943447071a.png)



You can see that if you use mg/dL, the grid is 50.
The lines in that case represent 50, 100, 150, 200, 250, 300 and 350.  

That translates to a grid size of 50 / 18 = 2.777 mmol/L.  That obviously is not as clean as 50.
The result is lines at: 2.78, 5.65, 8.33, 11.1, 13.9, 16.67 and 19.44.
However, what is shown on screen is: 2.8, 5.5, 8.3, 11, 14, 17 and 20.

It's not going to be easy to make a reasonably accurate estimate of the levels looking at those lines and those values if you use mmol/L.

**Comment by Navid200** [comment]
This has already been addressed for the distribution graph.  What remains is the AGP graph.  I will add that as well later.

**Comment by Navid200** [comment]
This should really be a discussion.
